src/20_Components/amp-bind.html

```diff
@@ -79,24 +92,36 @@
 
   `[property]="expression"`
 
-  In this example, the `src` of this &lt;amp-img&gt; element will change when
-  either the `allAnimals` or `currentAnimal` implicit state variables change.
+-->
 
-  - `allAnimals` is the `id` of the &lt;amp-state&gt; component above.
-  - `currentAnimal` is an uninitialized implicit state variable, which will be
-     set when a state change is triggered (see below).
-  -->
+<!-- #### Binding Text -->
+<!--
+The text of this &lt;p&gt; will change when the `currentAnimal` variable
+changes. `currentAnimal` is an uninitialized implicit state variable, which will be
+   set when a state change is triggered (see below).
+
+-->
+<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+
+<!-- #### Binding Styles -->
+<!--
+  Style class can be changed. It will override the existing element style class. `allAnimals` is the `id` of the &lt;amp-state&gt; component above.
```

Comment:
s/Style class can be changed./Styling can be changed by applying css classes./
s/the existing element style class/all the element's style classes/